### PR TITLE
Add Windows instructions for Git Appraise alias

### PR DESCRIPTION
**Pull request by TomasHubelbauer**

### CODE REVIEW

**Comment by TomasHubelbauer** [comment]
Closes #78

**Comment by TomasHubelbauer** [comment]
@googlebot I signed the CLA.

**Comment by ojarjur** [comment]
Thanks for taking the time and putting in the effort to help improve the user experience here!

This overall looks good to me, but I think the markdown formatting is a little off due to the 4 spaces before the `Windows` subtitle.

File: README.md [hunk 1]
```diff
@@ -27,6 +27,8 @@ Then, either make sure that `${GOPATH}/bin` is in your PATH, or explicitly add t
 "appraise" git alias by running the following command.
 
     git config --global alias.appraise '!'"${GOPATH}/bin/git-appraise"
+    # Windows:
```
Comment (ojarjur) [inline README.md]: I think this needs to be three lines like:

```

#Windows

```

... and then the subtitle will be formatted the way you want.

Comment (TomasHubelbauer) [reply README.md]: I actually meant to make that display like a Bash comment not a MarkDown heading but I'm happy to change it to that!

**Pull request merged**